app/views/checklist_tests/_field_values.html.erb

```diff
@@ -3,21 +3,102 @@
 # local variables
 #
 #   field_values   (Hash)
+#   product_test (ChecklistTest)
```

Comment:
Should add other locals. As far as I can tell there are two others: `negation` and `result`